types/src/keyless/circuit_constants.rs

```diff
@@ -55,26 +55,26 @@ pub fn devnet_prepared_vk() -> PreparedVerifyingKey<Bn254> {
 
     let delta_g2 = g2_projective_str_to_affine(
         [
```

Comment:
Why the VK change?

Reply:
This reflects the latest vk that should be loaded at genesis.  I updated via governance but didnt update genesis (because its such a pain in the ass to do)

Reply:
So this is actually the same VK I generated for the VK rotation test... 

I guess the `trusted-setup.sh` script does not use any randomness.